modules/gapi/src/api/freetype_mask_creator.hpp

```diff
@@ -31,10 +31,18 @@ class FreeTypeBitmaskCreator : public IBitmaskCreator
 public:
     FreeTypeBitmaskCreator(const std::string& path) : m_font_path(path)
     {
+        if (FT_Init_FreeType(&m_library) ||
+            FT_New_Face(m_library, m_font_path.c_str(), 0, &m_face))
+        {
+            m_is_initiliazed = false;
```

Comment:
Noo, please.

Don't construct object (== `throw`) if the initialization has failed. Just it

Reply:
Fixed